wandb/sdk/internal/internal_api.py

```diff
@@ -869,6 +912,48 @@ def update_run_queue_item_warning_introspection(self) -> bool:
         _, _, mutations = self.server_info_introspection()
         return "updateRunQueueItemWarning" in mutations
 
+    def _check_server_feature(self, feature_value: ServerFeature) -> bool:
+        """Check if a server feature is enabled.
+
+        Args:
+            feature_value (ServerFeature): The enum value of the feature to check.
+
+        Returns:
+            bool: True if the feature is enabled, False otherwise.
+
+        Raises:
```

Comment:
i don't think we raise exception anymore?

Reply:
oh i guess the query can raise an exception? in that case should we just raise the error here, whey do we need another wrapper function ?